java/HealthVault Mobile/HVSample/src/main/java/com/microsoft/hsg/android/hvsample/MainActivity.java

```diff
@@ -55,7 +56,10 @@ protected void onCreate(Bundle savedInstanceState) {
 		}
 
 		mHVClient = new HealthVaultClient();
-		
+
+		// TODO: Do we need the bottom navigation? I need to talk to designers.
```

Comment:
if we don't have a need for this can we just remove it? Or at least create a work item to track this so it doesn't get lost.